external/include/stress_tool/st_perf.h

```diff
@@ -200,6 +200,15 @@ void perf_set_keeprunning(int enable);
 		}                                                               \
 	} while (0)
 
+#define ST_ASSERT_NEQ(val, exp)                                          \
+	do {                                                                \
+		if (exp == val) {                                               \
```

Comment:
Macro name is NEQ, not equal, but if `exp` and `val` is equal, it makes assert.
Is this intended?